src/types.ts

```diff
@@ -160,8 +160,9 @@ export interface SessionRecordingOptions {
     recorderVersion?: 'v1' | 'v2'
     recordCrossOriginIframes?: boolean
     /** Modify the network request before it is captured. Returning null stops it being captured */
-    // TODO this has to work for both capture mechanisms? 😱
+    // deprecated - use maskCapturedNetworkRequestFn instead
     maskNetworkRequestFn?: ((data: NetworkRequest) => NetworkRequest | null | undefined) | null
```

Comment:
note to self: if someone has the configured then we should respect it, we'd only act on URL but that's better than suddenly ignoring provided config.

If someone configures this and `maskCapturedNetworkRequestFn` then that's an error. We should definitely log and possibly throw